test/integration/test-video-players-helper.js

```diff
@@ -65,22 +65,24 @@ export function runVideoPlayerIntegrationTests(
         const pauseButton = createButton(r, 'pause');
         const muteButton = createButton(r, 'mute');
         const unmuteButton = createButton(r, 'unmute');
+        // Dailymotion doesn't support mute/unmute when video is paused,
```

Comment:
If that's the case, we can't support daily motion as autoplay on mobile only works if video is muted, so before we call `play`, `mute` must be honored. Might be worth investigating to see if there is any workaround for this.